practices/coordination.md

```diff
@@ -12,48 +12,94 @@ Here are the goals we optimize for in organizing our team coordination practices
-4. **A collection of deliverables**. What are the deliverables that must exist in order to complete the project? These can start off more abstract, but over time they should become more concrete, and ultimately, lead to actionable tasks.
+The Project Backlog contains all of the deliverables across our projects that we wish to work on quickly.
+These are organized into a few columns, representing the _state_ of each deliverable.
+Here is a common column structure:
+
+- {guilabel}`Needs Discussion/Refinement`: Deliverables that are high-priority but un-refined. Our goal should be having discussion and doing research in order to get these deliverables ready for work.
+- {guilabel}`Ready to Work`: Deliverables that are well-scoped and have a clear path forward, and are thus ready to implement. As deliverables in {guilabel}`In progress` are completed, we should replace them with deliverables from this column. Generally speaking, deliverables near the top have higher priority than those at the bottom.
+- {guilabel}`In progress`: Deliverables that we are currently working towards. This means that we should be generating [Tasks](coordination:tasks) in our [Active Development Board](https://github.com/orgs/2i2c-org/projects/5) to complete this deliverable (more on this below).
+- {guilabel}`Blocked/Waiting`: Deliverables for which we are waiting for some action that is out of our team's immediate control.
+- {guilabel}`Done`: Deliverables that have been completed. We should close these issues and celebrate the improvements that we have made!
 
 :::{note}
-Deliverables are generally written as **user stories** - that is, with a particular user in mind, and a particular outcome they want to see.
-For example, "As a 2i2c Hub User, I want to install a package on my own."
-Deliverables should be accomplishable in one or two sprints, and may have many tasks associated with them.
+In general, there should be only two or three deliverables per team member, per column on this board.
+It should not become so heavily-populated that it is hard to keep track of deliverables!
 :::
 
-## Tasks are steps to complete a deliverable
+There are two Project Backlogs at 2i2c:
 
-In order to meet a deliverable, we need to take a set of actions, called **Tasks**.
-Tasks are verbs, not outcomes, and they should be extremely concrete and obvious what needs to be done.
-In addition, they should be accomplishable in one setting.
+- The [2i2c Development Projects Backlog](https://github.com/orgs/2i2c-org/projects/7) contains deliverables for our development-focused projects.
+- The [Organizatipnal Project Backlog](https://github.com/2i2c-org/meta/projects/6?fullscreen=true) contains deliverables that are organization-wide and not related to development.
```

Comment:
Maybe remove the full-screen from this last link? Or actually, add that to the previous link. The idea is to stay consistent.

Reply:
@choldgraf I still see the discrepancy here, even after your :+1: